docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -245,7 +245,8 @@ def digest_of(tag)
       end
 
       def fetch_digest_of(tag)
-        docker_registry_client.digest(docker_repo_name, tag)&.delete_prefix("sha256:")
+        manifest = docker_registry_client.manifest(docker_repo_name, tag)
```

Comment:
So it seems like the `GET` request to this endpoint ends up counting towards rate-limits whereas the `HEAD` request didn't, and since we make a lot of requests from a relatively small set of IP addresses, a lot of our updates ended up failing when I ran this in production :(